src/main/java/com/dataloom/data/DataApi.java

```diff
@@ -23,19 +30,27 @@
     String GET_DATA_PATH = "getData";
 
     String SET_ID        = "setId";
-    String SYNC_ID        = "syncId";
+    String SYNC_ID       = "syncId";
     String SET_ID_PATH   = "{" + SET_ID + "}";
-    String SYNC_ID_PATH   = "{" + SYNC_ID + "}";
+    String SYNC_ID_PATH  = "{" + SYNC_ID + "}";
+
+    String FILE_TYPE     = "fileType";
 
     @GET( CONTROLLER + "/" + ENTITY_DATA + "/" + SET_ID_PATH )
-    Iterable<SetMultimap<FullQualifiedName, Object>> getEntitySetData( @Path( SET_ID ) UUID entitySetId );
+    Iterable<SetMultimap<FullQualifiedName, Object>> getEntitySetData(
```

Comment:
This may cause confusion with the `createEntityData` endpoint, since they both take two UUID's although one is GET and another is POST. What do you think if I just make the entityId an optional request param instead?